torchvision/utils.py

```diff
@@ -97,6 +97,7 @@ def save_image(tensor, filename, nrow=8, padding=2,
         **kwargs: Other arguments are documented in ``make_grid``.
     """
     from PIL import Image
+    tensor = tensor.clone()
```

Comment:
Instead of cloning the tensor, I agree with [@fmassa](https://github.com/pytorch/vision/issues/1206#issuecomment-518830836) that replacing the in-place operations is the better choice here, since it is unclear why they are used in-place in the first place.

Reply:
I see, let me change the PR

Reply:
done